.github/ci_commit_pins/xla.txt

```diff
@@ -1 +1 @@
-307af4313d2b0b0236618ef837959a41068cc272
+2f1b8ffa18e1c376f51bb62deea0f545b4f9cbee
```

Comment:
I just reabse my PR at https://github.com/pytorch/xla/pull/3983.

And here is updated hash: 06c3224a7ecfaf268434fd56c2eb3f9bc973f223.

Reply:
thanks. updated the PR.

Reply:
I rebased again to see if we can fix the build error. Here is the new hash: 743d9fd4d2311fdd514caa977da2b7f29948258d.

Reply:
Thanks @alanwaketan, I updated the xla.txt in this PR